.github/workflows/ci.yaml

```diff
@@ -18,25 +18,13 @@ jobs:
             -   name: "Checkout"
                 uses: actions/checkout@v2
 
-            -   name: "Set up Python 3.7"
-                uses: actions/setup-python@v1
-                with:
-                    python-version: '3.7' # Semantic version range syntax or exact version of a Python version
-
-            -   name: "Display Python version"
-                run: python -c "import sys; print(sys.version)"
-
-            -   name: "Install Sphinx dependencies"
-                run: sudo apt-get install python-dev build-essential
-
-            -   name: "Install Sphinx"
-                run: pip install --user sphinx
-
-            -   name: "Install custom requirements via pip"
-                run: pip install -r _build/.requirements.txt
+            -   name: "Build Docker image"
```

Comment:
to take a leaf from @localheinz's book
> This step isn't actually necessary, but it makes it obvious that the Docker image needs to be build.
> Otherwise, the build time will be attributed to the next step (where the Docker image is used).